common/main/ja.xml

```diff
@@ -7126,6 +7126,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<percentSign>%</percentSign>
 			<plusSign draft="contributed">+</plusSign>
 			<minusSign draft="contributed">-</minusSign>
+			<approximatelySign>約 </approximatelySign>
```

Comment:
I suspect you should trim this (and any others that have leading or trailing whitespace).

Reply:
Glad you noticed!

I kept the non-breaking space here because we want the output to be `約 100` and there is no algorithm to add a space between the sign and the number.  Since the sign is only going to be interpolated into pattern strings that are also specified in this file, it seemed safe to add a space.